content/codespaces/managing-codespaces-for-your-organization/choosing-who-owns-and-pays-for-codespaces-in-your-organization.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Choosing who owns and pays for codespaces in your organization
-shortTitle: 'Billing and ownership'
+shortTitle: 'Billing and Ownership'
 intro: 'You can choose whether codespaces are paid for and owned by your organization or by your members.'
-permissions: 'Organization owners can change an organization''s billing details and control who owns and pays for codespaces.'
+permissions: 'Organization owners can change an organization's billing details and control who owns and pays for codespaces.'
```

Comment:
The first apostrophe is required as an escape, and removing it is causing the test failure in your PR. We can switch to double quotes to make the source more readable by preventing the need for the escape.

```suggestion
permissions: "Organization owners can change an organization's billing details and control who owns and pays for codespaces."
```